src/legacy/core_plugins/kibana/public/management/sections/index_patterns/edit_index_pattern/tabs/tabs.tsx

```diff
@@ -50,7 +50,7 @@ interface TabsProps extends Pick<RouteComponentProps, 'history' | 'location'> {
 }
 
 const filterAriaLabel = i18n.translate('kbn.management.editIndexPattern.fields.filterAria', {
-  defaultMessage: 'Filter',
+  defaultMessage: 'Filter field types',
```

Comment:
Looks like we're using this both for a search field and now for a select field too. Just want to double check that the same label is correct for both of them?

Reply:
@myasonik addressed the labels. Thanks for prompting the question. The default filler for search bar is "Search" now, aria-label: "search fields", and for the drop down: "filter field types". Thanks!